doc/development-context-aware-applications/how-to-perform-geo-located-queries.md

```diff
@@ -1,6 +1,9 @@
 One very powerful feature in Context Broker GE is the ability to perform
 geo-located queries. You can query entities located inside (or outside)
-a region defined by a circle or a polygon.  
+a region defined by a circle or a polygon. 
+
+Remember to add the `fiware-service: tourguide` header to every request to use the NiceEating data. 
```

Comment:
The header `fiware-service: tourguide' must be added to every request, so that the proper tenant is specified.